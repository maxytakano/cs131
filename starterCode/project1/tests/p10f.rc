float a[10][5][2];
float b[10];
float c[2][10][3];
float d[5][4][3][2][1];

float e[2];
float e[a[9][5]];
float f[true];
float e[2] ;
float g [ true == false];
float h [3 < 4];
float i [3 | 4];

float j [3][2][4][false];

float k [3][3.3][false][2];
