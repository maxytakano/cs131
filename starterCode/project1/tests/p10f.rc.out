Error, "/tmp/cs131_testrunner_XyTaux/file.rc", line 7: 
  Index value of 5 is outside legal range [0,5).
Error, "/tmp/cs131_testrunner_XyTaux/file.rc", line 7: 
  redeclared identifier 'e'.
Error, "/tmp/cs131_testrunner_XyTaux/file.rc", line 8: 
  Index expression type (bool) in array declaration not equivalent to int.
Error, "/tmp/cs131_testrunner_XyTaux/file.rc", line 9: 
  redeclared identifier 'e'.
Error, "/tmp/cs131_testrunner_XyTaux/file.rc", line 10: 
  Index expression type (bool) in array declaration not equivalent to int.
Error, "/tmp/cs131_testrunner_XyTaux/file.rc", line 11: 
  Index expression type (bool) in array declaration not equivalent to int.
Error, "/tmp/cs131_testrunner_XyTaux/file.rc", line 14: 
  Index expression type (bool) in array declaration not equivalent to int.
Error, "/tmp/cs131_testrunner_XyTaux/file.rc", line 16: 
  Index expression type (float) in array declaration not equivalent to int.
Compile: failure.
